### PR TITLE
Adds recommendation of SHA Pinning to example workflows

### LINKED ISSUE #17365: [Improvement]: Add comment to workflow examples that pin actions to SHAs
### Is this issue for a product update or release that is not yet available to customers?

No? Then please continue. Thank you!

### What article(s) is affected?

All workflow examples that pin actions to SHAs

### Describe the desired docs changes and the problem they solve for customers.

As a follow up to [this comment](https://github.com/github/docs/pull/15240/files#r801484042) and [this thread](https://github.slack.com/archives/CPU5KKSLQ/p1644353275298209):

We recommend pinning actions to SHAs for security reasons, but users sometimes don't realize that they need to update the SHA in our workflow examples to get the most up-to-date action.

We can add a comment like this to our workflow examples that pin actions to SHAs:

```
GitHub recommends pinning actions to a commit SHA. To get a newer version, you will need to update the SHA. You can also reference a tag or branch, but the action may change without warning.
```

**Note: We should make this a reusable so that we can easily add and change as needed. For an example, see usage of the `{% data reusables.actions.actions-not-certified-by-github-comment %}` reusable. (`data/reusables/actions/actions-not-certified-by-github-comment.md`)**

### Who does this affect?

actions users

### What is the impact to users?

confusion about why an action doesn't work as expected

### Are you willing to open a PR to make this fix? If not, do you know anyone (Hubber or not, we're open source!) who's willing to help write this documentation?

na

### PR DESCRIPTION
**Pull request by thispsj**
### Why:

Fixes #17365 

### What's being changed:

Created a reusable containing following text : 
```GitHub recommends pinning actions to a commit SHA. To get a newer version, you will need to update the SHA. You can also reference a tag or branch, but the action may change without warning.```

For content directory changes refer to the table below made by @github-actions.

### Check off the following:

- [X] I have reviewed my changes in staging.
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by Valerianastasia — COMMENTED** [review]
Leave a review

**Comment by thispsj** [comment]
@ramyaparimi ig this is ready for a review. ✨

**Comment by ramyaparimi** [comment]
@thispsj Thanks for the ping. I will get this triaged for a review ✨ .

**Review by meetsussie — COMMENTED** [review]
Good ideas

**Comment by jaystyles** [comment]
@dependabot merge

File: content/actions/automating-builds-and-tests/building-and-testing-java-with-gradle.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ You can also add this workflow manually by creating a new file in the `.github/w
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-java-with-gradle.md:53]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/automating-builds-and-tests/building-and-testing-python.md [hunk 2]
```diff
@@ -379,6 +379,7 @@ For this example, you will need to create two [PyPI API tokens](https://pypi.org
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-python.md:382]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/automating-builds-and-tests/building-and-testing-ruby.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ To get started quickly, add the starter workflow to the `.github/workflows` dire
 
 ```yaml
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-ruby.md:40]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/automating-builds-and-tests/building-and-testing-ruby.md [hunk 4]
```diff
@@ -101,6 +102,7 @@ The full updated workflow with a matrix strategy could look like this:
...
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-ruby.md:105]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/automating-builds-and-tests/building-and-testing-ruby.md [hunk 5]
```diff
@@ -238,6 +241,7 @@ The following example installs `rubocop` and uses it to lint all files. For more
...
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-ruby.md:244]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/automating-builds-and-tests/building-and-testing-ruby.md [hunk 6]
```diff
@@ -205,6 +207,7 @@ The following example matrix tests all stable releases and head versions of MRI,
...
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-ruby.md:210]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/automating-builds-and-tests/building-and-testing-ruby.md [hunk 7]
```diff
@@ -264,6 +268,7 @@ You can store any access tokens or credentials needed to publish your package us
...
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-ruby.md:271]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/automating-builds-and-tests/building-and-testing-swift.md [hunk 8]
```diff
@@ -68,7 +68,10 @@ The examples below demonstrate using the `fwal/setup-swift` action.
 You can configure your job to use multiple versions of Swift in a matrix.
 
 ```yaml{:copy}
+
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/automating-builds-and-tests/building-and-testing-swift.md:73]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-amazon-elastic-container-service.md [hunk 9]
```diff
@@ -92,6 +92,7 @@ Ensure that you provide your own values for all the variables in the `env` key o
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-amazon-elastic-container-service.md:95]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-docker-to-azure-app-service.md [hunk 10]
```diff
@@ -80,6 +80,7 @@ Ensure that you set `AZURE_WEBAPP_NAME` in the workflow `env` key to the name of
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-docker-to-azure-app-service.md:83]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-java-to-azure-app-service.md [hunk 11]
```diff
@@ -66,6 +66,7 @@ Ensure that you set `AZURE_WEBAPP_NAME` in the workflow `env` key to the name of
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-java-to-azure-app-service.md:69]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-net-to-azure-app-service.md [hunk 12]
```diff
@@ -65,6 +65,7 @@ Ensure that you set `AZURE_WEBAPP_NAME` in the workflow `env` key to the name of
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-net-to-azure-app-service.md:68]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: data/reusables/package_registry/publish-docker-image.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline data/reusables/package_registry/publish-docker-image.md:3]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions.md [hunk 14]
```diff
@@ -89,10 +89,12 @@ Create a new workflow file in your repository (such as `.github/workflows/deploy
 {% data reusables.package_registry.publish-docker-image %}
 
 {% else %}
-```yaml{:copy}
-name: Create and publish a Docker image
 
+```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions.md:95]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/issues/trying-out-the-new-projects-experience/automating-projects.md [hunk 15]
```diff
@@ -63,6 +63,7 @@ For more information about other changes you can make to your project through th
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/issues/trying-out-the-new-projects-experience/automating-projects.md:66]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/publishing-packages/publishing-java-packages-with-gradle.md [hunk 16]
```diff
@@ -224,7 +228,9 @@ publishing {
 With this configuration, you can create a workflow that publishes your package to both the Maven Central Repository and {% data variables.product.prodname_registry %} by running the `gradle publish` command.
 
 ```yaml{:copy}
+
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/publishing-packages/publishing-java-packages-with-gradle.md:233]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/publishing-packages/publishing-java-packages-with-gradle.md [hunk 17]
```diff
@@ -146,7 +148,9 @@ publishing {
 With this configuration, you can create a workflow that publishes your package to {% data variables.product.prodname_registry %} by running the `gradle publish` command.
 
 ```yaml{:copy}
+
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/publishing-packages/publishing-java-packages-with-gradle.md:153]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/publishing-packages/publishing-java-packages-with-gradle.md [hunk 18]
```diff
@@ -77,7 +77,9 @@ publishing {
 With this configuration, you can create a workflow that publishes your package to the Maven Central Repository by running the `gradle publish` command. In the deploy step, you’ll need to set environment variables for the username and password or token that you use to authenticate to the Maven repository. For more information, see "[Creating and using encrypted secrets](/github/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)."
 
 ```yaml{:copy}
+
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/publishing-packages/publishing-java-packages-with-gradle.md:82]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/publishing-packages/publishing-docker-images.md [hunk 19]
```diff
@@ -193,6 +196,7 @@ The following example workflow uses the steps from the previous sections ("[Publ
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/publishing-packages/publishing-docker-images.md:199]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/publishing-packages/publishing-docker-images.md [hunk 20]
```diff
@@ -142,8 +143,10 @@ The `build-push-action` options required for {% data variables.product.prodname_
 The above workflow is triggered by a push to the "release" branch. It checks out the GitHub repository, and uses the `login-action` to log in to the {% data variables.product.prodname_container_registry %}. It then extracts labels and tags for the Docker image. Finally, it uses the `build-push-action` action to build the image and publish it on the {% data variables.product.prodname_container_registry %}.
 
 {% else %}
+
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/publishing-packages/publishing-docker-images.md:149]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/publishing-packages/publishing-docker-images.md [hunk 21]
```diff
@@ -68,6 +68,7 @@ The `build-push-action` options required for Docker Hub are:
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/publishing-packages/publishing-docker-images.md:71]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/migrating-to-github-actions/migrating-from-circleci-to-github-actions.md [hunk 22]
```diff
@@ -408,6 +408,7 @@ workflows:
 
 ```yaml
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/migrating-to-github-actions/migrating-from-circleci-to-github-actions.md:411]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/managing-issues-and-pull-requests/scheduling-issue-creation.md [hunk 23]
```diff
@@ -31,7 +31,8 @@ In the tutorial, you will first make a workflow file that uses the [`imjohnbo/is
 
     ```yaml{:copy}
 {% indented_data_reference reusables.actions.actions-not-certified-by-github-comment spaces=4 %}
-
+{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```
Comment (skedwards88) [inline content/actions/managing-issues-and-pull-requests/scheduling-issue-creation.md:34]:
```suggestion

{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-nodejs-to-azure-app-service.md [hunk 24]
```diff
@@ -71,6 +71,7 @@ Ensure that you set `AZURE_WEBAPP_NAME` in the workflow `env` key to the name of
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-nodejs-to-azure-app-service.md:74]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-php-to-azure-app-service.md [hunk 25]
```diff
@@ -65,6 +65,7 @@ Ensure that you set `AZURE_WEBAPP_NAME` in the workflow `env` key to the name of
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-php-to-azure-app-service.md:68]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-python-to-azure-app-service.md [hunk 26]
```diff
@@ -68,6 +68,7 @@ Ensure that you set `AZURE_WEBAPP_NAME` in the workflow `env` key to the name of
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-python-to-azure-app-service.md:71]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md [hunk 27]
```diff
@@ -54,6 +54,7 @@ This workflow uses the `helm` render engine for the [`azure/k8s-bake` action](ht
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-kubernetes-service.md:57]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-static-web-app.md [hunk 28]
```diff
@@ -53,6 +53,7 @@ For more information about these values, see "[Build configuration for Azure Sta
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-azure/deploying-to-azure-static-web-app.md:56]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-google-kubernetes-engine.md [hunk 29]
```diff
@@ -136,6 +136,7 @@ Under the `env` key, change the value of `GKE_CLUSTER` to the name of your clust
 
 ```yaml{:copy}
 {% data reusables.actions.actions-not-certified-by-github-comment %}
+{% data reusables.actions.actions-use-sha-pinning-comment %}
```
Comment (skedwards88) [inline content/actions/deployment/deploying-to-your-cloud-provider/deploying-to-google-kubernetes-engine.md:139]:
```suggestion

{% data reusables.actions.actions-use-sha-pinning-comment %}
```

File: content/actions/managing-issues-and-pull-requests/adding-labels-to-issues.md [hunk 30]
```diff
@@ -31,6 +31,7 @@ In the tutorial, you will first make a workflow file that uses the [`andymckay/l
 
     ```yaml{:copy}
 {% indented_data_reference reusables.actions.actions-not-certified-by-github-comment spaces=4 %}
+{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```
Comment (skedwards88) [inline content/actions/managing-issues-and-pull-requests/adding-labels-to-issues.md:34]:
```suggestion

{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```

File: content/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added.md [hunk 31]
```diff
@@ -32,6 +32,7 @@ In the tutorial, you will first make a workflow file that uses the [`peter-evans
 
     ```yaml{:copy}
 {% indented_data_reference reusables.actions.actions-not-certified-by-github-comment spaces=4 %}
+{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```
Comment (skedwards88) [inline content/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added.md:35]:
```suggestion

{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```

File: content/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards.md [hunk 32]
```diff
@@ -33,6 +33,7 @@ In the tutorial, you will first make a workflow file that uses the [`alex-page/g
 
     ```yaml{:copy}
 {% indented_data_reference reusables.actions.actions-not-certified-by-github-comment spaces=4 %}
+{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```
Comment (skedwards88) [inline content/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards.md:36]:
```suggestion

{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```

File: content/actions/managing-issues-and-pull-requests/removing-a-label-when-a-card-is-added-to-a-project-board-column.md [hunk 33]
```diff
@@ -30,28 +30,28 @@ In the tutorial, you will first make a workflow file that uses the [`andymckay/l
...
+          runs-on: ubuntu-latest{% ifversion fpt or ghes > 3.1 or ghae or ghec %}
+          permissions:
+            issues: write
+            pull-requests: write{% endif %}
+          steps:
+            - name: remove labels
+              uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
+              with:
+                remove-labels: "needs review"
+                repo-token: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
```
Comment (skedwards88) [inline content/actions/managing-issues-and-pull-requests/removing-a-label-when-a-card-is-added-to-a-project-board-column.md:54]: it looks like this got extra indentation

```suggestion
    name: Remove labels
    on:
      project_card:
        types:
          - moved
    jobs:
      remove_labels:
        if: github.event.project_card.column_id == '12345678'
        runs-on: ubuntu-latest{% ifversion fpt or ghes > 3.1 or ghae or ghec %}
        permissions:
          issues: write
          pull-requests: write{% endif %}
        steps:
          - name: remove labels
            uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
            with:
              remove-labels: "needs review"
              repo-token: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
```

File: content/actions/managing-issues-and-pull-requests/removing-a-label-when-a-card-is-added-to-a-project-board-column.md [hunk 34]
```diff
@@ -30,28 +30,28 @@ In the tutorial, you will first make a workflow file that uses the [`andymckay/l
...
-        permissions:
-          issues: write
-          pull-requests: write{% endif %}
-        steps:
-          - name: remove labels
-            uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
-            with:
-              remove-labels: "needs review"
-              repo-token: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
+{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```
Comment (skedwards88) [inline content/actions/managing-issues-and-pull-requests/removing-a-label-when-a-card-is-added-to-a-project-board-column.md:35]:
```suggestion

{% indented_data_reference reusables.actions.actions-use-sha-pinning-comment spaces=4 %}
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for adding this! I'm going to add a new line before the comment for readability, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**